src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -81,16 +81,16 @@ internal sealed class TaintIssuesSynchronizer : ITaintIssuesSynchronizer
 
         public async Task SynchronizeWithServer()
         {
-            var bindingConfiguration = configurationProvider.GetConfiguration();
-
-            if (IsStandalone(bindingConfiguration) || !IsConnected() || !IsFeatureSupported())
-            {
-                HandleNoTaintIssues();
-                return;
-            }
-
             try
             {
+                var bindingConfiguration = configurationProvider.GetConfiguration();
+
+                if (IsStandalone(bindingConfiguration) || !IsConnected() || !IsFeatureSupported())
```

Comment:
When I saw the exception, the call to `IsFeatureSupported` was the one that failed.